lib/studies/fortnightly/app.dart

```diff
@@ -11,7 +11,7 @@ import 'package:gallery/layout/text_scale.dart';
 import 'package:gallery/l10n/gallery_localizations.dart';
 import 'package:gallery/studies/fortnightly/shared.dart';
 
-const fortnightlyTitle = 'Fortnightly';
+const _fortnightlyTitle = 'Fortnightly';
```

Comment:
Should this be removed and use the value from demos.dart instead?

Reply:
It'd be a bit cumbersome to obtain so I'll leave it as is